ynr/settings/base.py

```diff
@@ -212,7 +212,7 @@ def root(*path):
 INTERNAL_IPS = ["127.0.0.1"]
 
 # Language settings (calculated above):
-LANGUAGES = [("en", "English")]
+LANGUAGES = [("en-gb", "British English")]
```

Comment:
Were this change required due to an upgrade?

Reply:
Yes, I believe it was the result of an error related to the django 3.0 upgrade.